src/core/lib/iomgr/error_internal.h

```diff
@@ -28,39 +28,4 @@
 
 #include "src/core/lib/iomgr/error.h"
 
-#ifndef GRPC_ERROR_IS_ABSEIL_STATUS
```

Comment:
Seems like there's nothing actually left in this file.  Might as well remove it.

Reply:
Yep, done!